pkg/util/mapper/mapper.go

```diff
@@ -83,6 +84,11 @@ type PrimitiveMapReader interface {
 	// GetBool parses a bool from the map key parameter. If the value
 	// is empty or fails to parse, the defaultValue parameter is returned.
 	GetBool(key string, defaultValue bool) bool
+
+	// GetList returns a string list which contains the value set by key split using the
+	// provided delimiter with each entry trimmed of space. If the value doesn't exist,
+	// nil is returned
+	GetList(key string, delimiter string) []string
```

Comment:
Should we call this `GetStringList` in case we want to leave room for `GetIntList` or something like that?

Reply:
Yeah... Thought about this as well, multiple times, changed my mind, then changed it again. The general resolution I landed on was that the underlying data provider is a string map `Get(key string) string`, and the `Mapper` `Get` implementation is: `Get(key string, defaultValue string) string`, so the specialization adds the qualifiers, and `string` is always the default "value" type.